src/components/toast/__stories__/ToastConsumer.tsx

```diff
@@ -83,8 +91,48 @@ export const ToastConsumer = () => {
             rows={5}
             value={message}
           />
+          <Select
+            label="Direction"
+            onChange={v => {
+              if (v) {
+                setDir(v);
+              }
+            }}
```

Comment:
```suggestion
            onChange={v => v && setDir(v)}
```
What do you think about this shorthand?